x-pack/plugins/actions/server/action_type_registry.ts

```diff
@@ -54,26 +63,35 @@ export class ActionTypeRegistry {
    */
   public ensureActionTypeEnabled(id: string) {
     this.actionsConfigUtils.ensureActionTypeEnabled(id);
+    // Important to happen last due to feature usage being notified at the same time
```

Comment:
nit: This comment seems unclear to me.  Seems like the comment should indicate that this should be last so we don't do the notification if the action type **isn't** enabled.  Or is there something else going on that I'm missing?

I guess the gist of this is that if this method got called and the action type isn't enabled, it shouldn't go into the license check since that would send a notification that it was used, when in fact it wasn't.  If so, good catch!

Reply:
>I guess the gist of this is that if this method got called and the action type isn't enabled, it shouldn't go into the license check since that would send a notification that it was used, when in fact it wasn't. If so, good catch!

Correct, I will clarify the comment, thanks!

Reply:
Fixed in https://github.com/elastic/kibana/pull/77679/commits/f9368fd8062992b2a9ffaddce2c0466667e55992.